src/script/conversation/ConversationVerificationStateHandler/MLS/MLSStateHandler.ts

```diff
@@ -143,6 +148,12 @@ class MLSConversationVerificationStateHandler {
       return;
     }
 
+    const conversationExists = await this.core.service?.mls?.conversationExists(conversation.groupId);
```

Comment:
If when verifying e2ei state, mls group doesn't exist anymore, reset to default (unverified)